rhombus/scribblings/ref-reducer-macro.scrbl

```diff
@@ -120,7 +120,7 @@
     expr.macro 'build_finish [$count, $_, $_, $_, $_, $finish, $data]':
       'values($finish $data, $count + 1)'
   ~repl:
-    for counted(List) (i: 0..3): i
+    for counted(PairList) (i: 0..3): i
```

Comment:
This one doesn’t need to change, I think.

Reply:
Right, the change to `List` exposed a limitation/bug in the example. Not fixed, yet.

Reply:
HF, since you fixed the example.